lib/src/acrcloud_response.dart

```diff
@@ -105,3 +108,26 @@ class ACRCloudResponseArtist {
   factory ACRCloudResponseArtist.fromJson(Map<String, dynamic> json) =>
       _$ACRCloudResponseArtistFromJson(json);
 }
+
+/// A single matching Custom File
```

Comment:
nit: end the sentence with a period.

https://dart.dev/guides/language/effective-dart/documentation#do-format-comments-like-sentences